specification/network/resource-manager/Microsoft.Network/stable/2021-02-01/privateEndpoint.json

```diff
@@ -909,6 +921,21 @@
           "type": "string",
           "description": "Name of the resource that is unique within a resource group. This name can be used to access the resource."
         },
+        "id": {
```

Comment:
This seems to be a breaking change, could you follow the breaking change review process for it?  http://aka.ms/bcforapi